engine/glfw/lib/x11/x11_joystick.c

```diff
@@ -296,6 +300,15 @@ static void pollJoystickEvents( void )
             // Read all queued events (non-blocking)
             while( read(_glfwJoy[i].fd, &e, sizeof(struct js_event)) > 0 )
```

Comment:
Interestingly, this code doesn't check if a joystick is ever disconnected 🤔 
I.e. no one has reported this issue (yet). Not sure GLFW has that feature either...